parser.hpp

```diff
@@ -174,6 +174,14 @@ namespace Sass {
       return peek< mx >(peek < css_comments >(start));
     }
 
+    // at-directives that aren't just prefixes
+    template <prelexer mx>
+    const char* peek_directive(const char* start = 0)
+    {
+      // now peek a token (skip comments first)
+      return peek< sequence< mx, negate< identifier > > >(start);
```

Comment:
Maybe it would be more correct to have a [word-boundary] [1] match (like regex has with `\b`)?
Other than that I fully agree that this should be a step in the right direction!

[1]: http://www.regular-expressions.info/wordboundaries.html